rust/decentraland-godot-lib/src/dcl/mod.rs

```diff
@@ -20,6 +20,10 @@ use std::{
 #[derive(Default, PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Copy, Debug)]
 pub struct SceneId(pub u32);
 
+impl SceneId {
+    pub const INVALID: u32 = 0xFFFF_FFFF;
```

Comment:
This is equal to -1 in i32. And I'm using the -1 to know if the scene is empty. Can we use it as `SceneId::EMPTY` instead of INVALID?